client/src/stores/history/historyItemsStore.js

```diff
@@ -82,8 +86,10 @@ export const useHistoryItemsStore = defineStore("historyItemsStore", {
                 payload.forEach((item) => {
                     if (item.state == "ok") {
                         const itemCreateTime = new Date(item.create_time);
-                        if (itemCreateTime > state.latestCreateTime) {
+                        if (!state.latestCreateTime || itemCreateTime > state.latestCreateTime) {
                             state.latestCreateTime = itemCreateTime;
+                        } else if (itemCreateTime > state.lastUpdateTime) {
+                            state.lastUpdateTime = itemCreateTime;
```

Comment:
Thank you for fixing this feature, it works well. Wondering if we can consolidate `lastUpdateTime` with `latestCreateTime` here since they both are set to the same value.